mlir/utils/performance/perfRunner.py

```diff
@@ -209,6 +209,26 @@ def getMilliseconds(output):
 
     return float(result.group(1))
 
+def runPipeline(proc_specs):
```

Comment:
Abstract the common idiom of making a chain of processes, waiting for the resuts, and handling errors and returns.  Also made a slight change to allow cases which print to stderr but with a zero return code to still complete normally -- some tests emit warnings that don't indicate a problem, for instance MIOpenDriver.